src/lib/local-storage-helpers.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * Helper functions for loading and saving graphs to/from local storage
+ */
+
+/**
+ * takes a graph object & graph name, stringifies, and saves to local storage
+ * in "saved" object with unique key of date string
+ */
+export const saveGraphToLocal = (graph, name, preview, frames, frameIDs) => {
+  let savedGraphs = localStorage.getItem('gifsmos-saved-graphs');
+  if (!savedGraphs) {
+    savedGraphs = {
+      saved: {}
+    };
+  } else {
+    savedGraphs = JSON.parse(savedGraphs);
+  }
+  const newGraph = {
+    name,
+    graph,
+    preview,
+    frames,
+    frameIDs
+  };
+  let timeStamp = new Date().toString();
```

Comment:
Perhaps consider `let timeStamp = new Date().toLocaleString()` instead? (format ~ "6/13/2019, 4:02:43 PM") The upside is that it saves you slicing/adjusting both upon creation and loading.